lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -953,6 +958,28 @@ ChangeResult TypeAnalyzer::visitAtenMaxPool2dOp(
   return getLatticeElement(op->getResult(0)).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAtenConstantPadNdOp(
+    AtenConstantPadNdOp op,
+    ArrayRef<LatticeElement<ValueKnowledge> *> operands) {
+  auto knowledge =
+      ValueKnowledge::getNotNonePessimisticValueState(op->getContext());
+  knowledge.hasSizes = true;
```

Comment:
this need to be deleted at this line as we might not have sizes information for the output.